test/parallel/test-zlib-from-gzip-with-trailing-garbage.js

```diff
@@ -19,7 +19,7 @@ zlib.gunzip(data, common.mustCall((err, result) => {
   assert.strictEqual(
     result.toString(),
     'abcdef',
-    'result should match original string'
+    `result '${result.toString()}' should match original string`
```

Comment:
The best would probably be to just remove the message overall, since we still do not know what "the original string" is.